plots/generic.py

```diff
@@ -0,0 +1,539 @@
+"""
+Simple but highly configurable plotting functions for spiketrains in neo format.
+While building on the matplotlib libary the functions lay an emphasis on clear,
+pleasant-to-look-at visualizations.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from elephant.statistics import mean_firing_rate
+from math import log10, floor
+
+
+def _round_to_1(x):
+    rounded = round(x, -int(floor(log10(abs(x)))))
+    return rounded, rounded > x
+
+
+def _get_attributes(spiketrains, key_list):
+    """
+    attribute_array is of shape (len(spiketrains), len(key_list))
```

Comment:
It does not become clear that `attribute_array` is the return of the function. I think this doc string could be a bit improved (although its just a hidden function)